src/main/java/com/google/gerrit/extensions/common/EditInfo.java

```diff
@@ -14,10 +14,13 @@
 
 package com.google.gerrit.extensions.common;
 
+import com.google.gson.annotations.SerializedName;
+
 import java.util.Map;
 
 public class EditInfo {
   public CommitInfo commit;
+  @SerializedName("baseRevision")
```

Comment:
The naming of `baseRevision` got changed to `base_revision` in Gerrit itself:
https://gerrit-review.googlesource.com/c/67956/

You can also see the updated JSON response for your test here:
https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#get-edit-detail

Does this also work with your version of Gerrit? In that case, SerializedName can be removed. Otherwise, it would be good to have two tests, one for the old response and one for the new one.